Gruntfile.js

```diff
@@ -97,11 +97,17 @@ module.exports = function(grunt) {
       tasks: 'dev'
     },
     connect: {
-      dev: {
+      preview: {
```

Comment:
Is there a way we can get all the watch tasks and the server into one
command?

-- 
Sent from mobile